test/fixtures/rules/std_rule_cases/L048.yml

```diff
@@ -0,0 +1,17 @@
+rule: L048
+
+test_pass_1:
+  # Test that we don't fail * operators in brackets
```

Comment:
I don't understand the comment here -- it doesn't seem to relate to the test query.

Reply:
oops yeah, that's a remnant from L006